javascript/testing_javascript/more_testing.md

```diff
@@ -67,13 +67,12 @@ There are two solutions to the 'tightly coupled code' problem.  The first, and b
 
 <div class="lesson-content__panel" markdown="1">
 
-1. If you haven’t already, watch [FunFunFunction's "Mocking Basics" video](https://www.youtube.com/watch?v=3PjdxjWK0F0).
+1. If you haven’t already, watch [about mocking basics](https://www.youtube.com/watch?v=ajiAl5UNzBU&t=3024s).
 1. Too much mocking can be a bad thing.  It *is* sometimes necessary, but if you have to set up an elaborate system of mocks to test any bit of your code, that means your code is too tightly coupled.  While it is quite in-depth, the following article contains several really good points about [program architecture and testing](https://medium.com/javascript-scene/mocking-is-a-code-smell-944a70c90a6a).
 1. Now that you have some practice and context for TDD, the [Jest docs section on "Setup and Teardown"](https://jestjs.io/docs/setup-teardown) will probably make good sense to you.
 1. Read about [Jest's really handy mocking functions](https://jestjs.io/docs/mock-functions).
 1. Watch this amazing video that covers [what to test in your codebase](https://www.youtube.com/watch?v=URSWYvyc42M).
-
-  The video is specifically about testing the Ruby language, but that doesn't matter *at all*.  The concepts here ring true in any language, and luckily Ruby is a clear enough language that you will be able to follow along just fine.
+   The video is specifically about testing the Ruby language, but that doesn't matter *at all*.  The concepts here ring true in any language, and luckily Ruby is a clear enough language that you will be able to follow along just fine.
```

Comment:
We didn't actually need to remove the blank line before this, as removing the blank line would stop it from being a new paragraph.

That being said, I actually think it looks better in the same paragraph rather than separate, which makes the single line break redundant (if it's the same paragraph, we should keep the sentence on the same line in the file).

Could you combine this line with the previous please? i.e. line 74 should now contain:
```md
1. Watch this amazing video that covers [what to test in your codebase](https://www.youtube.com/watch?v=URSWYvyc42M). The video is specifically about testing the Ruby language, but that doesn't matter *at all*. The concepts here ring true in any language, and luckily Ruby is a clear enough language that you will be able to follow along just fine.
```